app/utils/profileManagement.js

```diff
@@ -0,0 +1,66 @@
+import { DISPLAY_NAME_FIELD } from 'containers/Profile/constants';
+
+import { saveText, getText, getImage } from './ipfs';
+import { getTableRow, sendTransaction } from './eosio';
+
+export async function uploadImageFile(image) {
+  const savedText = await saveText(image);
```

Comment:
I think we need new method in ipfs module to save binary content. saveText should be used for saving textual data, not images